styles/globals.css

```diff
@@ -33,23 +38,26 @@
   h6 {
     @apply text-base;
   }
-
-.border-outset {
+  /*Focused*/
+  h1:focus {
+    @apply ring-1;
+  }
+  .border-outset {
     border-style: outset;
-}
+  }
 
   
 /* 
 
-  below is the CSS from Canada theme 
+  below is the CSS from Canada theme =============================================================
 
 */
 
 
-.header {
+/* .header {
   background-color: #26374a;
   color: white;
-}
+} */

```

Comment:
Can we get rid of this in addition to all the other commented out code in `globals.css`?